workshop/lab1.3_configure.md

```diff
@@ -0,0 +1,123 @@
+# Setup
+
+As a preqrequisite to running these labs, you will need access to an OpenShift v4.x cluster as a cluster admin.  We also assume you have the `oc` client installed and have logged into your cluster with it.
+
+
+## Operator Installation
+
+In OpenShift, installation of Knative is very easy.  You just need to find and install the OpenShift Serverless Operator.
+
+![OpenShift Serverless Operator Hub](content/images/serverless_operator_hub.png)*Operator Hub*
+
+Then click install.
+
+![Install OpenShift Serverless Operator](content/images/serverless_install_operator.png)*OpenShift Serverless Operator*
+
+Next make sure to install in all namespaces.  Select the latest Update Channel made available to you.  For the purposes of this demo, we'll pick the Manual approval strategy.  Lastly, click Subscribe.
+
+![Subscribe to OpenShift Serverless Operator](content/images/serverless_create_subscription.png)*Subscribe*
+
+Wait a few minutes and eventually you will see it show up as Installed Operator with the status of Succeeded
+
+![Subscribe to OpenShift Serverless Operator](content/images/serverless_installed_operator.png)*Installed*
+
+
+## Knative Serving Installation
+
+1.  Create the `knative-serving` namespace
+```
+oc create namespace knative-serving
+```
+
+2.  Create a `serving.yaml`
+```
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+    name: knative-serving
+    namespace: knative-serving
+```
+
+3.  Apply the `serving.yaml` file
+```
+oc apply -f serving.yaml
+```
+
+4.  Verify installation
+```
+oc get knativeserving.operator.knative.dev/knative-serving -n knative-serving --template='{{range .status.conditions}}{{printf "%s=%s\n" .type .status}}{{end}}'
+```
+
+Wait a while and the output should look like the following
+```
+DependenciesInstalled=True
+DeploymentsAvailable=True
+InstallSucceeded=True
+Ready=True
+```
+
```

Comment:
Can we add a "verify pods" step similar to knative eventing?

Reply:
done